.editorconfig

```diff
@@ -80,3 +80,6 @@ trim_trailing_whitespace = unset
 
 [pkgs/tools/misc/timidity/timidity.cfg]
 trim_trailing_whitespace = unset
+
+[pkgs/applications/editors/vim/plugins/vim-command-check-hook.sh]
+indent_style = unset
```

Comment:
Can't this be done in the file? e.g.

https://github.com/NixOS/nixpkgs/blob/69e2ca68be5716e0873a99ce208d1e2635cad98e/nixos/tests/systemd-networkd-vrf.nix#L164

https://github.com/editorconfig-checker/editorconfig-checker#excluding

Reply:
thanks didn't know about that, that did it ;)